includes/Core/Authentication/Authentication.php

```diff
@@ -1359,4 +1360,27 @@ private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
 		return $feature_enabled;
 	}
 
+	/**
+	 * Invalid nonce error handler.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $action Action name.
+	 */
+	public static function invalid_nonce_error( $action ) {
+		if ( strpos( $action, 'googlesitekit_proxy_' ) !== 0 ) {
+			wp_nonce_ays( $action );
+			return;
+		} else {
```

Comment:
This is redundant, we don't need this `else` here. Let's get rid of it.

Reply:
https://github.com/google/site-kit-wp/pull/3976/commits/72855cff21aa68eb4977874409f63f5ca88292a4